butterfree/load/writers/writer.py

```diff
@@ -50,3 +54,41 @@ def validate(
             AssertionError: if validation fails.
 
         """
+
+    def build(
+        self, feature_set: FeatureSet, dataframe: DataFrame, spark_client: SparkClient,
```

Comment:
I think the name `build` here on a `Writer` sounds odd. But I don't have any other good call too. Maybe `load`, giving an idea of starting a loading action of the writer, that consists of running the pre_hooks and finally writing. Another thought is if the method `write` should now be private (e.g. `_write`) to not confuse the user in what method he should use. In my opinion, users should only use this method from now on.

Reply:
Maybe @roelschr and @AlvaroMarquesAndrade could provide some thoughts here too.

Reply:
`load` would be a nice name :+1:

Reply:
`load` seems better, I also had problems with the name "build" hahah